CleanArchitecture/Infrastructure/PersonItem.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using CleanArchitecture.Domain;
+using Microsoft.Azure.CosmosRepository;
+using Microsoft.Azure.CosmosRepository.Attributes;
+
+[Container("people")]
+[PartitionKeyPath("/id")]
+public class PersonItem : EtagItem
+{
+    public string Name { get; set; }
```

Comment:
```suggestion
    public string Name { get; set; } = null!;
```

Just to get rid of the null warnings here.